ui/base/ime/input_method_minimal_unittest.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/base/ime/input_method_minimal.h"
+
+#include "base/macros.h"
+#include "base/memory/singleton.h"
```

Comment:
do you really need this singleton.h include?

Reply:
Sorry, I removed it.